Makefile

```diff
@@ -533,23 +535,27 @@ $(foreach _type, $(BUILD_TYPES), $(eval $(call PUBLISH_SIGNED_RULE,$(_type))))
 
 define MSI_RULE
 .PHONY: msi-$(1)
-msi-$(1): ## create signed exe(s) and msi in $(BUILD_DIR)/signed
+msi-$(1): ## create signed exe and msi files in $(BUILD_DIR)/signed
 	$(eval BUILD_DIR := ${TARGET_DIR}/$(TARGET)/$(1))
 	$(eval SRC_ROOT := $(CURDIR))
-	$(eval CERT_FILE_NAME := "mezmo_cert_$(1).pfx")
+	$(eval HOST_TRIPLET := $(shell gcc -dumpmachine))
```

Comment:
What's the purpose of this eval statement? `HOST_TRIPLET` isn't used anywhere.

Reply:
leftover. removing. thanks